datafeed_scripts/avinetworks/vs_status_script.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+"""
+AVI ControlScript that updates the `UP` meta of an NS1 Datafeed.
+
+Invoked by the AVI Controller when a "VS_UP" or "VS_DOWN" alert is raised
+(alert action configured as ControlScript) and updates `UP` meta accordingly.
+
+`datasource_id` and `ns1_apikey` global vars must be configured in order to
+update the datafeed.  `ns1_endpoint` can optionally be configured to the FQDN
+of a custom API endpoint if using Private DNS or DDI.
+"""
+import argparse
+import json
+import sys
+
+from ns1 import Config, NS1
+
+# ID of the datasource in NS1 to publish updates to
+datasource_id = "a53252f9e583c6708331a1daeb172e12"
+# NS1 apikey to use when publishing updates
+ns1_apikey = "qACMD09OJXBxT7XOuRs8"
+# FQDN of NS1 API endpoint.  Leave as is if using Managed DNS.
+ns1_endpoint = "api.nsone.net"
+
+
+def _init_api():
```

Comment:
why the leading underscore everywhere? seems kinda weird to do in a simple script like this

Reply:
No good reason, just force of habit from working on another project.  I'm with you that it's prob weird in this context and down with doing away with it.